include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -5276,8 +5276,8 @@ namespace internal
   template <int dim, typename Number>
   struct EvaluatorTensorProduct<evaluate_general,dim,-1,0,Number>
   {
-    static const unsigned int dofs_per_cell = numbers::invalid_unsigned_int;
-    static const unsigned int n_q_points = numbers::invalid_unsigned_int;
+    static const unsigned int dofs_per_cell = 0;
+    static const unsigned int n_q_points = 0;
```

Comment:
That seems like a regression. The previous version clearly says "we don't know what dofs_per_cell is", whereas the new version suggests that we may know.

Is there no way downstream to only use the value of `dofs_per_cell` in case we determine that it is a valid value?